.env

```diff
@@ -46,3 +46,4 @@ REACT_APP_USER_SERVICE_CURRENT=/api/v1/users/current/
 REACT_APP_USER_SERVICE_LOGOUT=/api/v1/users/logout/
 
 REACT_APP_STATUS_SERVICE=/api/v1/status/
+REACT_APP_AUTH_TOKEN_SERVICE=/api/v1/token/
```

Comment:
It seems like there's a missing newline at the end of this line/file (indicated by the the red ⛔  symbol in github).

Not a big deal, but it'll make future commits and diffs cleaner since most editors include the newline.

Reply:
updated!